src/applications/find-va-forms/components/Card.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import classnames from 'classnames';
+
+export default function Card({ heading, href, description }) {
+  const containerClassName = classnames(
+    'vads-u-display--block',
+    'vads-u-background-color--primary-alt-lightest',
+    'vads-u-padding--2',
+    'vads-u-height--full',
+    'vads-u-text-decoration--none',
+  );
+
+  return (
+    <a href={href} className={containerClassName}>
+      <div className="vads-u-font-family--serif vads-u-font-weight--bold vads-u-text-decoration--underline">
+        {heading}
+      </div>
+      <div className="vads-l-gird-container">
+        <div className="vads-l-row">
+          <div className="vads-l-col--3 vads-u-margin-y--1">
+            <hr className="vads-u-margin--0 vads-u-border-color--cool-blue-lighter" />
+          </div>
+        </div>
+      </div>
+      <p className="vads-u-color--base">{description}</p>
+    </a>
+  );
+}
```

Comment:
@rtwell  I heard that this is some talk of a new Design System component called a Card. Is that what the blue boxes in this Find VA Forms page are? I wrote this so that it could be easy to port to Formation if we want to.

Reply:
We've had some discussion on eBenefits about a card component being added to formation as well. Not sure how much research has gone into what it should look like, but we created a ticket a little while ago for this very thing: 
https://github.com/department-of-veterans-affairs/va.gov-team/issues/2627

Reply:
Awesome! I'll post into ticket so we can see that once there is a Design System component, this might be an area that should also be updated.